.github/workflows/pr_enclave_tls.yml

```diff
@@ -127,8 +129,8 @@ jobs:
     - name: Test enclave tls server for mutual nullquote
       run: |
         echo random-port3 ${{ steps.random-port-generator3.outputs.random-port }}
-        docker exec $inclavare_test bash -c 'cd /opt/enclave-tls/bin/ && ./enclave-tls-server -a nullquote -m -p ${{ steps.random-port-generator3.outputs.random-port }}' &
-        docker exec $inclavare_test bash -c 'sleep 10 && cd /opt/enclave-tls/bin/ && ./enclave-tls-client -l debug -a nullquote -m -p ${{ steps.random-port-generator3.outputs.random-port }}'
+        docker exec $inclavare_test bash -c 'cd /usr/share/enclave-tls/samples && ./enclave-tls-server -a nullquote -m -p ${{ steps.random-port-generator3.outputs.random-port }}' &
```

Comment:
Use `/usr/share/enclave-tls/samples/enclave-tls-server` directly.

Reply:
Type `/usr/share/enclave-tls/samples/enclave-tls-server` does not work.
Unless add the command `cp /usr/share/enclave-tls/samples/sgx_stub_enclave.signed.so ./`

```
# /usr/share/enclave-tls/samples/enclave-tls-server
    - Welcome to Enclave-TLS sample server program
[ERROR] Failed to load enclave 8207
[ERROR] Failed to load sgx stub enclave
```